folding-schemes/src/folding/mova/mod.rs

```diff
@@ -0,0 +1,127 @@
+use crate::commitment::CommitmentScheme;
```

Comment:
Since this PR implements the Mova's NIFS but the NIFS methods and structs are 'not in use' in the eyes of clippy (eg. used in the IVC); to avoid clippy complaining about unused structs and methods, I think it's fine to add the `#![allow(unused)]` on the top of this file.

```suggestion
#![allow(unused)]
use crate::commitment::CommitmentScheme;
```